src/lib.rs

```diff
@@ -477,6 +470,15 @@ fn get_paths(env_variable_name: &str) -> impl Iterator<Item = String> + '_ {
                 .collect::<Vec<_>>()
         })
 }
+#[cfg(not(target_os = "wasi"))]
+use env::split_paths;
```

Comment:
```suggestion
use std::env::split_paths;
```
I think indirect use here doesn't have any advantage.